modules/op-components.adoc

```diff
@@ -0,0 +1,113 @@
+
+[id="op-overview-clustertasks_{context}"]
+== Cluster tasks and cluster trigger bindings
+
+When you install {pipelines-shortname}, the installation creates a certain number of `ClusterTask` and `ClusterTriggerBinding` CRs. These resources represent pre-arranged tasks and trigger bindings that you can use in your pipelines.
+
+Cluster tasks and cluster trigger bindings are deprecated and will be removed in a future release. Instead, you can configure the hub resolver to access a wide array of reusable components.
+
+[id="op-overview-chains_{context}"]
+== {tekton-chains}: signed storage of runs for security audit
+
+Any automatic process involving data from repositories is a potential security issue, because a repository might be compromised.
+
+To monitor and mitigate any possible security issues, {pipelines-shortname} provides {tekton-chains}, a method of storing the full contents of every task run and pipeline run, including any data (such as mounted volumes) that was processed in the run. This storage is signed by a key, so that any tampering with the storage is evident.
+
+You can use {tekton-chains} to set up security attestation for your CI/CD processes and to help implement the link:http://slsa.dev/[SLSA] guidelines.
+
+[id="op-overview-results_{context}"]
+== Tekton Results: access to execution data
+
+:FeatureName: Tekton Results
+include::snippets/technology-preview.adoc[]
+
+The full execution data of every task run and pipeline run, including run specs and logs, is stored in the `TaskRun` and `PipelineRun` CRs and can be signed and stored in a repository with {tekton-chains}. However, you might need to access these results in a more flexible manner to enable statistics and grouping. Also, `TaskRun` and `PipelineRun` resources can be pruned, while access to execution data and results might be required after the pruning.
+
+{pipelines-shortname} provides Tekton Results, which is an API and command line interface for querying execution and result data in a flexible way.
+
+[id="op-overview-pac_{context}"]
+== {pac}: running a pipeline from Git source
+
+With the {pac} functionality, you can define your CI/CD pipeline in your Git source code repository.
+
+Inside a directory in your source code, you define a template for the `PipelineRun` resource. This template includes the pipeline with all the necessary tasks. In addition, you register the repository with {pipelines-shortname} by using a CR and configure Git event notification using a webhook. If you use a GitHub repository, the {pac} installation routine can create a GitHub application for your repository that implements the notification.
+
+When events that you define in the template happen, {pipelines-shortname} runs the CI/CD pipeline defined in the source code.
+
+You can maintain this definition in your code repository, making changes as necessary, without any modification to {pipelines-shortname} resources.
```

Comment:
I love the change and rewrite. Super Misha.
Keep this together. Else, when you say 'this` definition, continuity is lost for the reader.